core/src/main/java/com/linecorp/armeria/common/logging/ClientConnectionTimingsBuilder.java

```diff
@@ -95,8 +95,10 @@ public ClientConnectionTimingsBuilder socketConnectEnd() {
      * in order to use one connection for HTTP/2.
      */
     public ClientConnectionTimingsBuilder pendingAcquisitionStart() {
-        pendingAcquisitionStartTimeMicros = SystemInfo.currentTimeMicros();
-        pendingAcquisitionStartNanos = System.nanoTime();
+        if (pendingAcquisitionStartTimeMicros == 0 && !pendingAcquisitionEndSet) {
```

Comment:
Could remove `!pendingAcquisitionEndSet`?

Reply:
I think we can't because `pendingAcquisitionStart*` will be overwritten

Reply:
Ah, please ignore this preview. 😉